content/docs/1.8.0/important-notes/_index.md

```diff
@@ -208,6 +210,18 @@ For more information, see [Issue #8184](https://github.com/longhorn/longhorn/iss
 
 Longhorn v1.8.0 and later versions support usage of V2 volumes in Talos Linux clusters. To use V2 volumes, ensure that all nodes meet the V2 Data Engine prerequisites. For more information, see [Talos Linux Support: V2 Data Engine](../advanced-resources/os-distro-specific/talos-linux-support#v2-data-engine).
 
+## Backup
+
+### Related Custom Resources Might Disappear
+
+Longhorn will attempt to clean up the backup-related custom resources in the following scenarios:
+
+- An empty response from the NFS server due to a brief server downtime.
+- A race condition between related Longhorn backup controllers.
```

Comment:
What does this mean?

Reply:
The detail of the race condition is in https://github.com/longhorn/longhorn/issues/9530#issuecomment-2378253692

Reply:
Just wondered if there is a better brief to describe the situation to make it understandable. For example, 

```
There is a race condition that could potentially delete the remote backup volume and its corresponding backups when resetting the backup target within a short period.
```

Also, can explain what backup-related custom resources are. For example, backup volume and backup, as the default backup target can only be cleaned up/reset by the operators.